tests/src/test/java/alluxio/master/JournalShutdownIntegrationTest.java

```diff
@@ -90,28 +93,65 @@ public final void before() throws Exception {
 
   @Test
   public void singleMasterJournalStopIntegration() throws Exception {
-    LocalAlluxioCluster cluster = setupSingleMasterCluster();
-    runCreateFileThread(cluster.getClient());
-    System.out.println(Configuration.get(PropertyKey.MASTER_JOURNAL_FOLDER));
-    // Shutdown the cluster
-    cluster.stopFS();
-    CommonUtils.sleepMs(TEST_TIME_MS);
-    awaitClientTermination();
-    reproduceAndCheckState(mCreateFileThread.getSuccessNum());
```

Comment:
don't you need to remove this method, since it is no more used?

Reply:
good catch, done